src/senselab/audio/tasks/speech_enhancement/speechbrain.py

```diff
@@ -75,10 +75,24 @@ def enhance_audios_with_speechbrain(
                     + " does not match expected "
                     + str(expected_sample_rate)
                 )
-
-            # Enhance waveforms in a batch
-            enhanced_waveform = enhancer.separate_batch(audio.waveform)
-
-            audio.waveform = enhanced_waveform.reshape(1, -1)
-
-        return audios
+        enhanced_audios = []
```

Comment:
hi @ibevers the problem with this code is that it doesn't work with files of different shape